### PR TITLE
Add sudo to update api

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/5805

### CODE REVIEW

File: ui/src/views/Settings.vue [hunk 1]
```diff
@@ -236,7 +236,7 @@ export default {
             console.error(e);
           }
         },
-        false
+        true; //true
```
Comment (edospadoni) [inline ui/src/views/Settings.vue]: If the param is not passed, the default is `true` so you can delete the entire line

File: ui/src/views/Settings.vue [hunk 2]
```diff
@@ -217,7 +217,7 @@ export default {
             function(error, data) {
               console.error(error, data);
             },
-            false
+            true; //sudo
```
[the remark this answers is not in the document]
Comment (edospadoni) [reply ui/src/views/Settings.vue]: you can check it here: https://github.com/NethServer/nethserver-cockpit/blob/master/root/usr/share/cockpit/nethserver/libs/nethserver.js#L3

File: ui/src/views/Settings.vue (hunk 1 above)
Comment (edospadoni) [reply ui/src/views/Settings.vue]: you can check it here: https://github.com/NethServer/nethserver-cockpit/blob/master/root/usr/share/cockpit/nethserver/libs/nethserver.js#L3

File: ui/src/views/Settings.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (stephdl) [reply ui/src/views/Settings.vue]: ok will do, but for a better code understanding, I would prefer to display the `true`

**Pull request merged**